st2common/st2common/models/system/action.py

```diff
@@ -206,7 +206,8 @@ def _get_script_arguments(self, named_args=None, positional_args=None):
 
         # add the positional args
         if positional_args:
-            command_parts.append(positional_args)
+            pos_args_string = ' '.join(map(quote_unix, positional_args))
```

Comment:
Any particular reason why you used map over a list comprehension?

In any case, LGTM.